# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: devtools/client/responsive.html/browser/tunnel.js
# Commit: 791382b2c9bf
# Full Hash: 791382b2c9bf60371b68c0dd4d2ed26e4b846901
# Author: Barret Rennie <barret@brennie.ca>
# Date: 2019-04-04 04:33:36
# Regressor Bug: 1510569
# File Overlap Count: 1
# Description:
#   Bug 1510569 - Port onStatusChange notifications inside WebProgressChild.jsm to C++ r=baku
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D25446
# ==============================================================================

diff -r 31b206e2046f -r 791382b2c9bf devtools/client/responsive.html/browser/tunnel.js
--- a/devtools/client/responsive.html/browser/tunnel.js	Wed Apr 03 17:31:41 2019 +0000
+++ b/devtools/client/responsive.html/browser/tunnel.js	Wed Apr 03 17:31:54 2019 +0000
@@ -433,7 +433,6 @@
     "Content:ProgressChange",
     "Content:SecurityChange",
     "Content:StateChange",
-    "Content:StatusChange",
     // Messages sent to browser.js
     "DOMTitleChanged",
     "ImageDocumentLoaded",